service/movie_service_impl.py

```diff
@@ -12,11 +12,16 @@ def __init__(self, client: MovieApiClient):
     def get_five_random_movies(self) -> list[Movie]:
         random_movies = []
         '''get one random movie per page'''
-        for page in range(1, 6):
+        for page in range(*self.__get_random_pages__()):
             movie = self.client.get_top_rated_movies(page)[self.__get_random_num__()]
             movie.set_image_url(self.client.get_image_url_by_movie_id(movie.get_id()))
             random_movies.append(movie)
         return random_movies
 
     def __get_random_num__(self):
-        return random.randint(0, 19)
\ No newline at end of file
+        return random.randint(0, 19)
+    
+    def __get_random_pages__(self) -> tuple:
+        '''MovieDB API includes 465 pages with 20 movies per page'''
+        start_page = random.randint(1, 460)
+        return (start_page, start_page + 5)
```

Comment:
So this might return pages for example: 300, 301, 302, 303, 304 if I understand this correctly?
Can we change this behaviour, so we can get pages for example 5, 60, 130, 290, 401?

Reply:
Yes, you're right. I'll add this in the next commit

Reply:
Here is the [fix](https://github.com/airetOK/movie-quizz-app/pull/5/commits/b5ed640a66b944f84db7ecef54beb5d584dfe9fb)